mqt/benchviewer/templates/description.html

```diff
@@ -74,31 +74,31 @@ <h1>MQT Bench: Level Description and File Format</h1>
       <p>Benchmarks are provided on four different abstraction levels:</p>
       <ul>
         <li>
-          <i>Algorithmic Level</i>: Circuits are described in a textbook-like
+          <em>Algorithmic Level</em>: Circuits are described in a textbook-like
           fashion using high-level constructs.
         </li>
         <li>
-          <i>Target-Independent Level</i>: Circuits are unfolded and unrolled
+          <em>Target-Independent Level</em>: Circuits are unfolded and unrolled
           representations of the circuits from the algorithmic level.
         </li>
         <li>
-          <i>Target-dependent Native-Gates Level</i>: Circuits are synthesized
+          <em>Target-dependent Native-Gates Level</em>: Circuits are synthesized
           to a particular native gate-set.
         </li>
         <li>
-          <i>Target-dependent Mapped Level</i>: Circuits are mapped to a
+          <em>Target-dependent Mapped Level</em>: Circuits are mapped to a
           particular architecture.
         </li>
       </ul>
       <p>Used File Format:</p>
       <ul>
         <li>
           Until openQASM 3.0 is widely supported, all circuits (except those on
-          the algorithmic level) are provided as <i>.qasm</i> files following
+          the algorithmic level) are provided as <em>.qasm</em> files following
           the openQASM 2.0 standard. Circuits on the algorithmic level can
           directly be generated (as IBM Qiskit QuantumCircuit objects) using the
-          <i>mqt.bench</i> python package. Instructions on how to use the
-          package are included in the downloaded <i>.zip</i> file whenever an
+          <em>mqt.bench</em> python package. Instructions on how to use the
```

Comment:
This sentence should be removed: "Instructions on how to use the package are included in the downloaded .zip file whenever an algorithmic level circuit is requested."

Reply:
Done in https://github.com/cda-tum/MQTBench/pull/136/commits/68e9d3ba959d52ec307e8bf69b4502eec5985d43